source/Reviewers/howtocommit.rst

```diff
@@ -514,6 +516,64 @@ for all affected tests before you commit to the trunk.
     failed for other reasons (e.g. timeout) then these should be re-triggered
     before attempting to install the KGO files.
 
+4.1 Managing BIG_DATA
+^^^^^^^^^^^^^^^^^^^^^^
+
+Static input data, such as ancilliaries, are required by many `lfric_apps` tests.
+
+LFRic's lfric_apps tests use a BIG_DATA_DIR environment variable to provide a platform based
+path prefix to provide direct access to data required for tests.
+
+The master copy of this is held on XCS at `/common/lfric/data/`.
+A `cron` job is run daily at 07:30 utc on `xcslr0` as the `lfric` user, which runs the script:
+
+https://github.com/MetOffice/lfric_tools/tree/main/bigData/rsyncBigData.sh
+
+from
+
+.. code-block:: RST
+
+    /home/d03/lfric/bigData/rsyncBigData.sh
+
+This script synchronises the content of `/common/lfric/data/` from `XCS` to `XCE/F` and `SPICE`,
+deleting all content not in `XCS` BIG_DATA from the remote locations and updating any changed content.
+
+This BIG_DATA_DIR is not versioned nor source controlled on any platform.
+Care is required. The ability to log in as the `lfric` user is required, e.g. via
+
+.. code-block:: RST
+
+    sudo -u lfric -i
+
+As reviewer, you should work with the developer to:
```

Comment:
a) this can probably be generallised a bit - "in charge of the trunk for the repositories involved" to cover all bases. 

b) I want to question/think about process here. Holding onto the trunk while waiting for the cron to run feels like it could be wasting peoples time. In the case where we're adding new files I think this is unneeded. Perhaps the following:
 * Place new files in the appropriate location (where that location is is per repo so should think about where/how to display that - or perhaps this whole section wants to be on per-repo tabs since the specifics of XCS first are also unique to LFRic)
 * Run relevant tests on XCS
 * Once the cron job has sychronised the data between s/ef/spice:
  * ensure you're in charge of the trunks
  * update your working copy if other commits have happened
  * run relevant tests on e/f and spice
  * proceed to commit
  
I think the existing files instructions is fine as with overwritting files you will want to make sure others aren't commiting and you've already suggested waiting for the cron to be a bad idea.

Reply:
Thinking about this further - assuming that per repo we've provided details further up on master location and synchronisation versions perhaps this could be generalised to:
* Place new files in the master location as described above
* Run tests on that platform
* Once the files have synchronised to all platforms:
  * Ensure you're in charge of the trunks
  * Update your working copy if other commits have happened
  * Run tests on all other platforms
  
  
  (I'm also not sure if "proceed to commit" is needed - I'm not sure we have "move to next step" on any other steps on this page).

Reply:
understood, i've reworded, to meet some of this, recognising that rewriting with UM in mind will occur